cmd/gctcli/commands.go

```diff
@@ -2805,3 +2806,112 @@ func clearScreen() error {
 		return cmd.Run()
 	}
 }
+
+const timeFormat = "2006-01-02 15:04:05"
+
+var startTime, endTime, order string
+var limit int
+
+var getAuditEventCommand = cli.Command{
+	Name:   "getauditevent",
+	Usage:  "gets audit events matching query parameters",
+	Action: getAuditEvent,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "start, s",
+			Usage:       "start date to search",
+			Value:       time.Now().Add(-time.Hour).Format(timeFormat),
+			Destination: &startTime,
+		},
+		cli.StringFlag{
+			Name:        "end, e",
+			Usage:       "end time to search",
+			Value:       time.Now().Format(timeFormat),
+			Destination: &endTime,
+		},
+		cli.StringFlag{
+			Name:        "order, o",
+			Usage:       "order results by ascending/descending",
+			Value:       "asc",
+			Destination: &order,
+		},
+		cli.IntFlag{
+			Name:        "limit, l",
+			Usage:       "how many results to retrieve",
+			Value:       100,
+			Destination: &limit,
+		},
+	},
+}
+
+func getAuditEvent(c *cli.Context) error {
+	if c.NArg() == 0 && c.NumFlags() == 0 {
```

Comment:
Since you set default values above and also check to see if the string != "" below, you can drop this so it will use the default values